src/main/java/Duke.java

```diff
@@ -12,20 +15,45 @@ public static void main(String[] args) {
         System.out.println("Hello from\n" + logo);
         System.out.println(line);
         System.out.println("Hello! I'm Duke\n" + "What can I do for you?");
```

Comment:
"\n" only works for Windows and does not work well on Mac and Linux. It will not achieve the intended effect of switching to the next line. In fact, Java provides a convenient line separator named "System.lineSeparator()".